lib/src/utils.dart

```diff
@@ -63,6 +63,12 @@ String _debugPrintError(String message, exception, stack) {
   return exceptionMessage;
 }
 
+String truncateWithEllipsis(int cutoff, String myString) {
+  return (myString.length <= cutoff)
+      ? myString
+      : '${myString.substring(0, cutoff)}...';
+}
+
```

Comment:
Could drop this again.

```suggestion
```

But if you want to keep it I would suggest an extension method:


```suggestion
extension TruncateWithEllipsis on String {
  String truncateWithEllipsis(int cutoff) => length <= cutoff ? this : "${substring(0, cutoff)}...";
}
```

By the way, what makes is it my string and not your string? 😄

Reply:
I decided to remove this snippet - better don't store code far a far future.